File: Codes/0047-permutations-ii_20250206-BuiltIn.py
'''
Author: LetMeFly
Date: 2025-02-06 13:55:07
LastEditors: LetMeFly.xyz
LastEditTime: 2025-02-06 13:56:30
'''
from typing import List
from itertools import permutations

class Solution:
    def permuteUnique(self, nums: List[int]) -> List[List[int]]:
        return list(set(permutations(nums)))

File: Codes/0047-permutations-ii_20250206-BuiltIn.cpp
/*
 * @Author: LetMeFly
 * @Date: 2025-02-06 13:52:38
 * @LastEditors: LetMeFly.xyz
 * @LastEditTime: 2025-02-06 13:53:41
 */
#ifdef _WIN32
#include "_[1,2]toVector.h"
#endif

class Solution {
public:
    vector<vector<int>> permuteUnique(vector<int>& nums) {
        vector<vector<int>> ans;
        sort(nums.begin(), nums.end());
        do {
            ans.push_back(nums);
        } while (next_permutation(nums.begin(), nums.end()));
        return ans;
    }
};